openff/evaluator/forcefield/forcefield.py

```diff
@@ -261,3 +261,43 @@ def __setstate__(self, state):
         self._cutoff = state["cutoff"]
         self._request_url = state["request_url"]
         self._download_url = state["download_url"]
+
+
+class FoyerForceFieldSource(ForceFieldSource):
+    """A wrapper around Foyer force fields"""
+
+    @property
+    def foyer_source(self):
+        """str: Foyer force field source."""
+        return self._foyer_source
+
+    @property
+    def cutoff(self):
+        """openff.evaluator.unit.Quantity: The non-bonded interaction cutoff."""
```

Comment:
Are these `openff.evaluator.unit.Quantity`s or `openff.units.unit.Quantity`s? The former is an alias of the latter, so it's not a big deal. I'm just curious what type they are presented as in this code

Reply:
Good question, I copied the doc string from the other forcefield sources which use that terminology

Reply:
Cool - it looks like all of the docstrings currently use `openff.evaluator.unit` so it's best to keep it that way for now. Consistency over accuracy, I guess